### ISSUE TITLE
support brackets in form_field post

### ISSUE THREAD

**Issue by cgrayson** [activeprospect/leadconduit-integration-custom#8]
if you set up mappings in a form like: `form_field.merge_vars[FNAME]` – in an attempt to get a form post body like: `…&merge_vars[FNAME]=#{first_name}&…` – you instead get posts with `merge_vars.FNAME=`.

**Comment by cgrayson** [comment]
This is happening [here](https://github.com/activeprospect/leadconduit-integration-custom/blob/master/src/normalize.coffee#L25).

E.g., 
```
coffee> l = require('lodash')
coffee> l.set {}, 'foo[bar]', 42
{ foo: { bar: 42 } }
```

**Comment by mrjuicebox** [comment]
Related to https://activeprospect.zendesk.com/agent/tickets/10661

**Comment by cgrayson** [comment]
Fixed in 2.8.4
